MerkleDistributionDiff.md

```diff
@@ -0,0 +1,60 @@
+The [AaveMerkleDistributor](./src/contracts/AaveMerkleDistributor.sol) contract is based on [Uniswap MerkleDistributor](https://github.com/Uniswap/merkle-distributor/blob/master/contracts/MerkleDistributor.sol).
+
+The logic has not changed, and both use [OpenZeppelin MerkleProof](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/cryptography/MerkleProof.sol) libs.
+
+The main differences between the two then is the addition of:
+
+- Token rescue methods:
+```
+function emergencyTokenTransfer(address erc20Token, address to, uint256 amount) external;
+function emergencyEtherTransfer(address to, uint256 amount) external;
+```
+
+- Distributions:
+The distributions is a system that enables our contract to support multiple rescues by adding new merkleTrees and tokens and encapsulating
```

Comment:
Important to highlight if the logic is just the same, and then we just add a mapping for multi-distribution.
Also important to describe how we did assumptions regarding multiple distributions of the same asset